packages/ssr/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "@toddledev/ssr",
-  "version": "0.0.2-alpha.3",
+  "version": "0.0.2-alpha.4",
```

Comment:
Have we discussed the option to run with a single version across all packages instead of individually? I think this is best for consumers, but we will have a tough time keeping track of which versions are compatible with each other.

Reply:
We should look into that eventually yes. Right now we only have 1 real consumer though, so I think it's fine to postpone to keep things simple and to avoid too much manual work when doing changes.